.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     "eqeqeq": "error",
     "indent": ["error", 2],
     "no-console": "off",
+    "no-unused-vars": ["error", {vars: "all", args: "after-used", ignoreRestSiblings: true, argsIgnorePattern: "options"}],
```

Comment:
This was added to address [as of yet] unused `...options` rest args, which I'm hoping to be able to use for custom configs and options.